performance/aws.py

```diff
@@ -1,4 +1,5 @@
 import boto3
+from performance import prod_config as config
```

Comment:
the naming `prod_config` is misleading but is off scope of this PR and I'll review it outside. Just putting a note here to record it, no action. ;)